### PR TITLE
List icon configurable

### PR DESCRIPTION
**Pull request by spi43984**
suggestion for #151

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -88,8 +88,11 @@ var $sitehead = $("#site-head");
       });
     }
 
-    $('ul').addClass("fa-ul");
-    $("ul li").prepend('<span class="fa-li"><i class="fa fa-asterisk"></i></span>');
+    var listElement = getComputedStyle(document.documentElement).getPropertyValue('--listElement');
+    if (listElement.length > 0) {
+      $('ul').addClass("fa-ul");
+      $("ul li").prepend('<span class="fa-li"><i class="fa ' + listElement + '"></i></span>');
```
Comment (zjedi) [inline assets/js/index.js:94]: if listElement itself is `fa-asterisk`, won't this compose into `fafa-asterisk`?

File: assets/js/index.js [hunk 2]
```diff
@@ -88,8 +88,11 @@ var $sitehead = $("#site-head");
       });
     }
 
-    $('ul').addClass("fa-ul");
-    $("ul li").prepend('<span class="fa-li"><i class="fa fa-asterisk"></i></span>');
+    var listElement = getComputedStyle(document.documentElement).getPropertyValue('--listElement');
+    if (listElement.length > 0) {
```
Comment (zjedi) [inline assets/js/index.js:92]: So if the property is undefined, there is no icon applied? Looks ok

File: assets/css/variables.scss [hunk 3]
```diff
@@ -38,4 +38,8 @@
     /* === Footer === */
     --footer-color-background: #22343a;
     --footer-color: #bbc7cc;
-}
\ No newline at end of file
+
+    /* === Lists === */
+    /* define listElement for unnumbered lists, leave empty for default listElement */
+    --listElement: fa-asterisk;
```
Comment (zjedi) [inline assets/css/variables.scss:44]: I'm thinking about the name ... how about `list-icon` or `li-icon`

**Review by zjedi — CHANGES_REQUESTED** [review]

File: assets/js/index.js (hunk 1 above)
Comment (spi43984) [reply assets/js/index.js:94]: I hope not... Originally there is `class="fa fa-asterisk"` which is then composed to  `class="fa ' + listElement + '"` - please note the space after the initial fa.

File: assets/js/index.js (hunk 2 above)
Comment (spi43984) [reply assets/js/index.js:92]: right, that is the idea. The default "fa fa-asterisk" gets defined in variables.scss and can be changed or omitted by the user.

File: assets/css/variables.scss (hunk 3 above)
Comment (spi43984) [reply assets/css/variables.scss:44]: li-icon sounds good to me - bu as this is for unnumbered lists only how about something like ul-li-icon? On the other hand numbered lists don't have any icons. So li-icon really sounds ok.

File: assets/js/index.js (hunk 1 above)
Comment (zjedi) [reply assets/js/index.js:94]: ah, ok, I missed the space

File: assets/css/variables.scss (hunk 3 above)
Comment (zjedi) [reply assets/css/variables.scss:44]: both ul-li-icon and li-icon sound ok to me :)

Comment (spi43984) [reply assets/css/variables.scss:44]: your code - you decide ;-)

Comment (zjedi) [reply assets/css/variables.scss:44]: ok, let's go with `ul-li-icon`, it's the most descriptive :)

**Comment by spi43984** [comment]
changes done - pls recheck

**Review by spi43984 — COMMENTED** [review]
changes done

**Review by zjedi — APPROVED** [review]

**Pull request merged**